app/Http/Requests/StoreUpdateValidateRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreUpdateValidateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'title' => 'required|string|max:255',
+            'description' => 'nullable|string|max:255',
+            'startedAt' => 'date|nullable',
+            'completedAt' => 'date|nullable',
+            'deadline' => 'date|nullable',
+            'status' => 'nullable|string|in:to do,in progress,review,completed',
```

Comment:
@Asatryan007 it is good practice to use not hard coded values. You can use constants.
For example instead of these values _to do,in progress,review,completed_ , use constants